src/main.py

```diff
@@ -179,6 +179,12 @@ async def on_message(message: discord.Message):
         log_info = parse_log(log_link)
         await message.channel.send(log_info)
 
+    for community_wiki_link in re.findall(r"https://stardewcommunitywiki.com/[a-zA-Z0-9_/:-%]+", message.content):
```

Comment:
I think the regex could be `https://stardewcommunitywiki.com/[a-zA-Z0-9_/:-%]*` (replacing the + with a *) as we also want to catch the home page as well.